client/my-sites/term-tree-selector/terms.jsx

```diff
@@ -100,6 +102,10 @@ const TermTreeSelectorList = React.createClass( {
 		if ( forceUpdate ) {
 			this.virtualScroll.forceUpdate();
 		}
+
+		if ( this.props.terms !== prevProps.terms ) {
+			this.recomputeRowHeights();
```

Comment:
No need to change here, but noticed in the `recomputeRowHeights` component function that we're calling both `recomputeRowHeights` and `forceUpdate` (the latter would be redundant and potentially affect performance).